my-app/src/styles/reset.css

```
@@ -1,44 +1,133 @@
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
```

Stylelint (17.11.0)

[error] Overridden property "font-size" by shorthand "font" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)